xonsh/ptk/shell.py

```diff
@@ -32,12 +32,22 @@ def __init__(self, **kwargs):
         self.prompter = Prompter()
         self.history = PromptToolkitHistory()
         self.pt_completer = PromptToolkitCompleter(self.completer, self.ctx)
-        self.key_bindings_manager = KeyBindingManager(
-            enable_auto_suggest_bindings=True,
-            enable_search=True,
-            enable_abort_and_exit_bindings=True,
-            enable_vi_mode=Condition(lambda cli: builtins.__xonsh_env__.get('VI_MODE')),
-            enable_open_in_editor=True)
+
+        key_bindings_manager_args = {
+                'enable_auto_suggest_bindings': True,
+                'enable_search': True,
+                'enable_abort_and_exit_bindings': True,
+                'enable_open_in_editor': True
+                }
+        import prompt_toolkit
+        vptk = getattr(prompt_toolkit, '__version__', '0.0')
+        major, minor = [int(x) for x in vptk.split('.')[:2]]
```

Comment:
Would you mind using `xonsh.shell.prompt_toolkit_version()` function, please? This duplicates that code to some degree. Thanks!